docs/docs/gatsby-style-guide.md

```diff
@@ -278,3 +281,21 @@ The Hemingway App will assign a “grade level” for your writing. You should a
 for a grade level of 6. Another tool available is the De-Jargonizer, originally
 designed for scientific communication but might help avoid overspecialized
 wording.
+
+## Best practices
+### Support software versions
+
+When Gatsby commits to support a specific version of software (e.g. Node 6 and up), this is reflected in documentation. Gatsby documentation should be usable by all people on supported software, which means we don't introduce any commands or practices that can't be used by people on versions we've committed to support. In rare circumstances, we'll consider mentioning a newly introduced commands and practices as side notes.
+
+For example, npm 5.2.0 (which includes Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, documentation should only introduce `npx` as an optional command in a note like so:
```

Comment:
> npm 5.2.0 (which includes Node 8)
`npm` gets installed when you install `Node` (not the other way around), so I think this would need to be re-worded a bit to be factually correct.